ui/main.qml

```diff
@@ -68,6 +69,15 @@ ApplicationWindow {
         property string locale: "en"
         property var recentEmojis: []
     }
+    Connections {
+        target: profileModel
+        onProfileSettingsFileChanged: {
+            settingsLoaded()
+            if (appSettings.locale !== "en") {
+                profileModel.changeLocale(appSettings.locale)
+            }
+        }
+    }
```

Comment:
@emizzle I updated like you said. Sadly, there is no `onSettingsChanged` or not even `on FileNameChanged` for the Settings, so I still needed a connection to the `profileModel` and update the language when the file changes